src/components/elements/Inputs/Map/RHFMap.tsx

```diff
@@ -46,6 +69,57 @@ const RHFMap = ({
       cacheTime: 0
     }
   );
+
+  const refetchData = () => {
+    reloadProjectPolygonData();
+    mapFunctions?.onCancel(polygonDataMap);
+  };
+
+  const {
+    data: projectPolygon,
+    refetch: reloadProjectPolygonData,
+    isRefetching
+  } = useGetV2TerrafundProjectPolygon(
+    {
+      queryParams: {
+        entityType: entity?.entityName || "",
+        uuid: entity?.entityUUID || ""
+      }
+    },
+    {
+      enabled: !!entity,
```

Comment:
Do you really want to be calling this API if entityName or entityUUID are null? Assuming not, I'd change this to
```
enabled: entity?.entityName != null && entity?.entityUUID != null
```
And then you could drop the default `?? ""` above because the API won't be called unless these are set.